cps/cp44.md

```diff
@@ -0,0 +1,100 @@
+---
+cp: 44
+title: Linked Pools
+authors: @mustermeiszer
+contributors:  @jeroen @akhan
+proposal-type: CP4 technical
+date-proposed: 2023-07-14
+date-ended: ---
+status: rfc
```

Comment:
Status will be **_polling_** (OpenSquare snapshot will be created)

Reply:
Done